db/people.yml

```diff
@@ -958,8 +958,8 @@ rpokratath:
       begin_month: 9
       degree: PhD in Chemistry
       institution: University of Basel
-      end_year: 2016
-      end_month: 4
+      end_year:
```

Comment:
do not leave blank entries.  ``end_year`` is not required, nor ``end_month``, so just delete completely these entries.